openwisp_firmware_upgrader/templates/admin/change_form_overriden.html

```diff
@@ -0,0 +1,16 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_modify %}
+
+{% block object-tools-items %}
+<li>
+    <form method="post" action="{% url 'admin:'|add:upgrade_url %}">
+        {% csrf_token %}
+        <input type="hidden" name="post" value="yes">
+        <input type="hidden" name="action" value="upgrade_selected">
+        <input type="hidden" name="_selected_action" value="{{object_id}}">
+        <input type="submit"  value="{% trans 'Upgrade Build' %}" style="border-radius:15px">
```

Comment:
On testing, this button looks out of place with `history` button.

![Qo8pYJ3](https://user-images.githubusercontent.com/24985760/86768089-9a89a600-c06a-11ea-83ef-4495fb2ed4df.png)

Maybe you can use `<a></a>` instead of `<input>`, which doesn't look correct.
If there is some reason why you chose `<input>`; maybe you can add these styles:
```
border-radius: 15px;
padding: 5px 15px;
background-color: #999;
font-weight: 400;
font-size: 13px;
line-height: 20px;
margin: 0;
```
and change `background-color: #333;` on hover.